app/webpacker/components/RegistrationsV2/api/payment/get/refundPayment.js

```diff
@@ -1,17 +1,24 @@
-import { fetchJsonOrError } from '../../../../../lib/requests/fetchWithAuthenticityToken';
+import { fetchWithAuthenticityToken } from '../../../../../lib/requests/fetchWithAuthenticityToken';
 import { refundPaymentUrl } from '../../../../../lib/requests/routes.js.erb';
 
 export default async function refundPayment({
   competitionId,
   paymentId,
   amount,
 }) {
-  return fetchJsonOrError(
+  return fetchWithAuthenticityToken(
```

Comment:
Why are you moving away from `fetchJsonOrError`?

Reply:
because the refund route doesn't return json ;)